cmake/modules/SIMDExt.cmake

```diff
@@ -49,6 +35,7 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64")
   # don't believe only the -march support; gcc 4.8.5 on RHEL/CentOS says
   # it supports +crc but hasn't got the intrinsics or arm_acle.h.  Test for
   # the actual presence of one of the intrinsic functions.
+  include(CheckCXXSourceCompiles)
```

Comment:
Why do we need to include this in this completely unrelated location?

Reply:
because i removed it from https://github.com/ceph/ceph/pull/24168/files#diff-4ceed8045270a39892752d02da901692L21 , and we need to use `check_cxx_source_compiles()` at https://github.com/ceph/ceph/pull/24168/files#diff-4ceed8045270a39892752d02da901692R40 . this function is offered by `CheckCXXSourceCompiles `, per https://cmake.org/cmake/help/v3.0/module/CheckCXXSourceCompiles.html

but this does not matter anymore, i don't think we need to check for the builtin function anymore. as we depends on GCC-7, which is trust worthy in this perspective.